crates/viewer/src/main.rs

```diff
@@ -75,8 +85,16 @@ impl GameApp for ViewerApp {
     }
 
     fn init(graphics_device: &mut GraphicsDevice) -> Self {
-        let example = Example::parse();
-        let shape = example.shape();
+        let args = AppArgs::parse();
+
+        let shape = if let Some(step_file) = args.step_file {
+            Shape::read_step(step_file).expect("Failed to read STEP file, {step_file}")
+        } else if let Some(example) = args.example {
+            example.shape()
+        } else {
+            println!("Warning - no example or STEP file specified, you get a default cube.");
```

Comment:
Consider using eprintln! instead to write to stderr.

```suggestion
            eprintln!("Warning - no example or STEP file specified, you get a default cube.");
```

Reply:
On another note: I am unable to resolve this conversation now (the resolve button is simply missing), but if you review any PRs, it is always present. Do you have an idea why this might by the case?

Reply:
I think it automatically marks it as resolved when a suggestion is committed to the branch from the github comment UI.